test/support/events/phoenix_multi_router.exs

```diff
@@ -1079,7 +1079,7 @@
     }
   },
   %{
-    event: [:phoenix, :endpoint, :stop],
+    event: [:internal, :endpoint, :stop],
```

Comment:
Do you mind sharing more about this change?

Reply:
Oh, I totally forgot about this PR. This was probably just for passing the test case, though I’m not sure. I will look into this again ASAP. Thanks for the comment. 

By the way, I made my custom phoenix plugin for this purpose and it has been working great for my team.

Reply:
> By the way, I made my custom phoenix plugin for this purpose and it has been working great for my team.

🚀 🥳  write some blog post or something see we learn about it 🚀 🥳

Reply:
After some investigation, here's what I've figured out.
Firstly, there are only two endpoints in the test file, as both endpoints define custom event_prefix (`[:external, :endpoint]`, `[:internal, :endpoint]`). So it seems like there should be only events starting with those prefixes in this file.

Next, in the phoenix_multi_router_test.exs, it looks like that it expect 2 `[:internal, :endpoint, :stop]` event for `"/really-cool-route"` that belongs to `additional_routes`. Before this change, there was only one event of it when it comes to the `[:internal, :endpoint, :stop]` event. 

So how the test could pass before? Well, in my guess, there is one more "/really-cool-route" path event, though it belongs to `[:external, :endpoint, :stop]`. I think it was caught with the wrong metadata before my changes.

I'm not 100% sure 'cause I didn't make that test originally but I think it would make sense..